pyqt5.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: pyqt5
+config-opts:
+- "--assume-shared"
+- "--concatenate"
+- "--confirm-license"
+- "--no-designer-plugin"
+- "--no-dist-info"
+- "--no-docstrings"
+- "--no-qml-plugin"
+- "--no-qsci-api"
+- "--no-stubs"
+- "--no-tools"
+- QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.7m/'
```

Comment:
Also, since the sip files are now generated, don’t forget to clean them up.

Reply:
done